### PR TITLE
Cleanup the first chapter

### PR DESCRIPTION
**Pull request by Gankro**
This is kinda WIP in that I haven't actually done a once-over of the changes. Interested in feedback before landing.

### CODE REVIEW

File: src/safe-unsafe-meaning.md [hunk 1]
```diff
@@ -6,37 +6,40 @@ interact?
...
 maintains the contracts the trait requires.
 
-You can use `unsafe` on a block to declare that all constraints required
-by an unsafe function within the block have been adhered to, and the code
-can therefore be trusted. You can use `unsafe` on a trait implementation
-to declare that the implementation of that trait has adhered to whatever
-contracts the trait's documentation requires.
+You can use `unsafe` on a block to declare that all unsafe actions performed
+within are verified to uphold the contracts of those operations. For instance,
+the index passed to `slice::get_unchecked` is in-bounds.
```
Comment (QuietMisdreavus) [inline src/safe-unsafe-meaning.md]: > For instance, *that* the index passed to [...]

(adding "that" there matches the structure both before and after)

File: src/safe-unsafe-meaning.md [hunk 2]
```diff
@@ -6,37 +6,40 @@ interact?
...
   memory safety to be freely violated.
 * `mem::transmute` reinterprets some value as having a given type, bypassing
   type safety in arbitrary ways (see [conversions] for details).
-* Every raw pointer to a sized type has an intrinstic `offset` method that
-  invokes Undefined Behavior if the passed offset is not "in bounds" as
-  defined by LLVM.
-* All FFI functions are `unsafe` because the other language can do arbitrary
-  operations that the Rust compiler can't check.
+* Every raw pointer to a sized type has an `offset` method that
+  invokes Undefined Behavior if the passed offset is not "in bounds".
```
Comment (Gankro) [inline src/safe-unsafe-meaning.md]: I'll link ptr::offset's docs (the answer is "uggggh llvm why")

**Comment by Gankro** [comment]
All reviews addressed, ready to land.

**Comment by steveklabnik** [comment]
https://travis-ci.org/rust-lang-nursery/nomicon/builds/260364075#L741

**Comment by Gankro** [comment]
Error unrelated, need @simonsapin's branch to be merged

**Comment by Gankro** [comment]
Fixed.

**Pull request merged**